src/newgrf_roadtype.cpp

```diff
@@ -65,16 +65,27 @@
 	return nullptr;
 }
 
+GrfSpecFeature RoadTypeResolverObject::GetFeature() const
+{
+	return GSF_ROADTYPES; // FIXME: when is this GSF_TRAMTYPES instead?
+}
+
+uint32 RoadTypeResolverObject::GetLocalID() const
+{
+	return this->roadtype_scope.rti->label;
```

Comment:
The local id is super weird to get (reverse lookup in road/tramtype_map), and is actually not used anywhere after initialisation. So I agree that "label" is more useful.
However, the method should not be named "GetLocalID" then.